content/graph.go

```diff
@@ -61,7 +61,12 @@ func Successors(ctx context.Context, fetcher Fetcher, node ocispec.Descriptor) (
 		if err := json.Unmarshal(content, &manifest); err != nil {
 			return nil, err
 		}
-		return append([]ocispec.Descriptor{manifest.Config}, manifest.Layers...), nil
+		var nodes []ocispec.Descriptor
+		if manifest.Subject != nil {
+			nodes = append(nodes, *manifest.Subject)
+		}
```

Comment:
`docker.MediaTypeManifest` should not have `manifest.Subject`.

Reply:
That is, the above statement

> // docker manifest and oci manifest are equivalent for successors.

is no longer true.

Reply:
fixed